corehq/apps/case_search/management/commands/compare_case_properties.py

```diff
@@ -0,0 +1,146 @@
+                        "lang": "groovy"
+                    }
+                }
+            ]
+        }
+    }
+    if case_type is None:
+        query_obj = (CaseSearchES()
+            .domain(domain)
+            .set_query(script))
+    else:
+        query_obj = (CaseSearchES()
+            .domain(domain)
+            .case_type(case_type)
+            .set_query(script))
+    return query_obj
+
+
+class Command(BaseCommand):
+    help = "Run test queries for perfomance benchmarks for comparison script queries."
+
+    def add_arguments(self, parser):
+        parser.add_argument('domain')
+        parser.add_argument('-ct', '--type', dest='type')
+        parser.add_argument('-cp', '--properties', dest='props', nargs=2)
+        parser.add_argument('-n', dest='n', default=1, type=int)
+
+        # operator arguments, only one of these is permitted
+        parser.add_argument('-eq', dest='equals', action='store_true')
+        parser.add_argument('-neq', dest='not_equals', action='store_true')
+        parser.add_argument('-gt', dest='greater_than', action='store_true')
+        parser.add_argument('-lt', dest='less_than', action='store_true')
+        parser.add_argument('-gte', dest='greater_equal', action='store_true')
+        parser.add_argument('-lte', dest='less_equal', action='store_true')
+
+        parser.add_argument('--verbose', dest="verbose", action='store_true')
+        parser.add_argument('--profile', dest="profile", action='store_true')
+
+    def handle(self, *args, **kwargs):
+        domain = kwargs['domain']
+        case_type = kwargs['type'] if kwargs['type'] else None
+        props = kwargs['props']
+        repeats = kwargs['n']
+        operators = {
+            "==": kwargs['equals'],
+            "!=": kwargs['not_equals'],
+            ">": kwargs['greater_than'],
+            "<": kwargs['less_than'],
+            ">=": kwargs['greater_equal'],
+            "<=": kwargs['less_equal']
+        }
+        operator_valuelist = list(operators.values())
+        operator_keylist = list(operators.keys())
+
+        if operator_valuelist.count(True) != 1:
+            raise CommandError('Only one operator flag may be invoked (-eq, -neq, -gt, -lt, -gte, -lte)')
+        if len(props) != 2:
+            raise CommandError("'properties' flag requires exactly 2 arguments")
+
+        operator = operator_keylist[operator_valuelist.index(True)]
+
+        # Need need to get baseline amount of cases that are being queriied against for context
+        # Define and run query with only domain and case_type defined
+        query_obj = define_baseline_query(domain, case_type)
+        baseline_amt = run_baseline_query(query_obj)
+
+        # Define, run, and analyze the test query
+        test_query_obj = define_test_query(domain, case_type, props, operator)
+        test_results, hits = run_test_queries(test_query_obj, repeats=repeats)
+        raw_took_str = f"TOOK: {query_obj.run().raw['took']}"
```

Comment:
`query_obj` here refers to the baseline query. Ideally we want to record the `took` value for each of the test queries that get run and compare it to the python timings (see comment above)

Reply:
Addressed here e2728f188585167d55868939c5688ca1f2c82928